readme.md

```diff
@@ -23,8 +23,16 @@ Get location data of any IP Address within seconds.
 
 ## 🎩 INSTALLATION
 
-Install the CLI globally. If you are windows user, run your command prompt/terminal as administrator. If you are macOS user, add `sudo` before the following command.
+Install the CLI globally. If you are windows user, run your command prompt/terminal as administrator.
 
+```
+npm i -g geo-location-cli
+```
+For Linux & macOS users
+```
+sudo su
+```
+Enter you password then following command below:
```

Comment:
The lingo is not right. Also, there is a typo. You can write something like this 

After this, type in your password and press Enter. Then copy-paste the following command in your terminal and install the CLI.